unit_tests/UnitTests.d

```diff
@@ -905,3 +905,21 @@ void foo ();
 D", options));
 
 }
+
+// Test global attributes
+unittest
+{
+    Options options;
+    options.globalAttributes = ["nothrow", "@nogc"];
+
+    assertTranslates(q"C
+void foo();
+C", q"D
+extern (C):
+nothrow:
+@nogc:
```

Comment:
In this case, yes. But then I thought it could also be used for other attributes like `package(...)`, or even `version(...)`, and that would make it less readable, but this might also be feature overuse...

I don't really care. I'm ok to change it.